rfcs/text/0044/host.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: host
+  fields:
+    - name: serial_number
```

Comment:
Should it be attribute of the host or device?

Reply:
I would say `host`